css/src/components/accordion.scss

```diff
@@ -37,10 +37,15 @@ $accordion-transition: transform 0.15s !default;
 	summary {
 		display: flex;
 		line-height: 1.5;
+		list-style: none;
 		cursor: pointer;
 		padding-block: $accordion-spacing;
 		gap: $accordion-gap;
 
+		&::-webkit-details-marker {
```

Comment:
Just to clarify, is the desired behavior to not show any chevron/marker at all, if for some reason the display is changed from flex to something else? If so, wouldn't we still need some sort of indicator to show that there's a toggle button for the accordion panels?

Reply:
Oh, we have a custom arrow there. Check out the screenshot, there are two icons but should be just one . 
In test steps I mentioned toggling the flex display rule cause it's the only way to see the difference before merging this PR

Reply:
I couldn't get both chevrons to show at the same time, but I see the `<marker>` appearing while we only want to show the `:before`.  Resolving.